### ISSUE TITLE
Spike - Custom picker

### ISSUE THREAD

**Issue by bessgreen** [department-of-veterans-affairs/va.gov-team#20779]
## Context for this task

The app currently uses the default picker styles for dropdowns. There are numerous accessibility & UI implications as a result of using the default style:

#### Android:

1.  UX/UI: the text in the dropdown is smaller than non-dropdown text. Because we're using the default style, we cannot adjust the font size. Related bug: #18615
2. UX/UI: When the picker is open, the focused state (i.e. the thicker blue border) will not appear on the dropdown field
3. UX/UI: If the user opens the picker but selects the placeholder instead of a value, they won't see the field error state on the picker field until they hit "Save".
4. Accessibility: _No known issues _

#### iOS:

1. Accessibility: The picker can be opened using voice control, but the user **cannot scroll** through the options by saying "scroll up" or "scroll down". Related bug: #18523
2. UX/UI: _No known issues_ 

## Dev task
Investigate options for creating a custom picker that does the following on both iOS & Android:

- [ ] Can be accessed/navigated with voice control software
- [ ] Font sizes are configurable 
- [ ]  Recognizes when the picker is open vs. closed, and as a result, would allow the app to:
     - [ ]  display a focused state, and 
     - [ ]  display an error state if it's opened/closed with no value selected

Follow-up task to build custom picker: #21131
